src/app/components/common/BorderScreen/BorderScreen.tsx

```diff
@@ -10,6 +10,7 @@ type Props = {
   SideHeader?: any,
   gray80?: boolean,
   overFlow?: string,
+  customSize?: number,
```

Comment:
Can we use just 'width' name instead of custom size?

Reply:
done